desktop/apps/consignments2/client/actions.js

```diff
@@ -151,17 +154,29 @@ export function createSubmission () {
       const {
         submissionFlow: {
           inputs,
+          submission,
           user
         }
       } = getState()
-      const token = await fetchToken(user)
-      const { body } = await request
-                          .post(`${sd.CONVECTION_APP_URL}/api/submissions`)
-                          .set('Authorization', `Bearer ${token}`)
-                          .send(inputs)
-      dispatch(updateSubmission(body)) // update state to reflect current submission
+      const token = await fetchToken(user.accessToken)
+
+      let submissionBody
+      if (submission.id) {
+        submissionBody = await request
+                           .put(`${sd.CONVECTION_APP_URL}/api/submissions/${submission.id}`)
+                           .set('Authorization', `Bearer ${token}`)
+                           .send(inputs)
+      } else {
+        submissionBody = await request
+                           .post(`${sd.CONVECTION_APP_URL}/api/submissions`)
+                           .set('Authorization', `Bearer ${token}`)
+                           .send(inputs)
+      }
```

Comment:
This seems kinda redundant, but there are enough differences to warrant it (url, http verb)